### ISSUE TITLE
adding other flags and separating the schema

### ISSUE THREAD

**Issue by chefallan** [warengonzaga/xplorescan#6]
- testnet and mainnet
- evm and non-evm

We will need to separate the schema for testnet and mainnet evm and non-evm chains to avoid latency searching on the schemas.

**Comment by chefallan** [comment]
cc: @warengonzaga

**Comment by warengonzaga** [comment]
Yes, this will make our scanning faster if seperate evm non evem or mainnet or testnet version of explorer.
